tests/openfl/utilities/test_verifiable_dataset.py

```diff
@@ -0,0 +1,332 @@
+import json
+from openfl.utilities.verifdataset.local_data_source import LocalDataSource
+from openfl.utilities.verifdataset.verifiable_dataset import DatasetFormat, VerifiableDataset
+import pytest
+import os
+from pathlib import Path
+from typing import List, Tuple
+
```

Comment:
I don't see where you import and use pyfakefs in the tests ?

Reply:
The fs fixture that is provided to the data_sources fixture.
AFAIK there no need to import anything